language/fr/lostpassword.lang

```diff
@@ -0,0 +1,39 @@
+<?php
+
+include("{$_EnginePath}language/en/lostpassword.lang");
```

Comment:
This file's translation is completed, therefore you can remove this default import.

Reply:
Done.